release-notes/MSqlServerServer/1.0/1.0.0.md

```diff
@@ -0,0 +1,42 @@
+# Release Notes
+
+## General Availability of Microsoft.SqlServer.Server
+
+_**1.0.0 released 28 January 2022**_
+
+This is the initial public stable release of the **Microsoft.SqlServer.Server** namespace in a separate assembly. This library is a helper for **Microsoft.Data.SqlClient** enabling cross framework support of UDT types.
+
```

Comment:
PR link is missing.

Reply:
This is the very beginning release. Following similar releases, we didn't include any PR!